docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -63,10 +63,14 @@ The following functionality will be removed in Ansible 2.14. Please update updat
 * :ref:`ec2_key <ec2_key_module>`: the ``wait`` option will be removed. It has had no effect since Ansible 2.5.
 * :ref:`ec2_key <ec2_key_module>`: the ``wait_timeout`` option will be removed. It has had no effect since Ansible 2.5.
 * :ref:`ec2_lc <ec2_lc_module>`: the ``associate_public_ip_address`` option will be removed. It has always been ignored by the module.
+* :ref:`iam_policy <iam_policy_module>`: the ``policy_document`` option will be removed. To maintain the existing behaviour use the ``policy_json`` option read the file using the ``lookup`` plugin
```

Comment:
```suggestion
* :ref:`iam_policy <iam_policy_module>`: the ``policy_document`` option will be removed. To maintain the existing behaviour use the ``policy_json`` option to read the file using the ``lookup`` plugin.
```

Reply:
@samccann How do you feel about this as an alternative:
```
To maintain the existing behavior use the ``policy_json`` option and read the file with the ``lookup`` plugin.
```

I think it's a little clearer?

Reply:
sounds good.